src/similar-npm-packages.ts

```diff
@@ -567,6 +597,16 @@ export const categories: Categories = {
     similar: ["raphael", "snapsvg", "two.js"],
   },
 
+  "template-engine": {
+    name: "Template engines",
```

Comment:
Let's call these "General purpose Templating Engines" (key too) to better differentiate them from HTML specific templating languages like "Pug".